src/Transfer/Worker.cpp

```diff
@@ -212,21 +212,124 @@ Worker::processCollisions(
   }
 
   for (int i = 0; i < numchares; i++) {
-    proxy[i].checkPoints(separated[i].size(), separated[i].data());
+    proxy[i].determineActualCollisions(separated[i].size(), separated[i].data());
```

Comment:
If I understand correctly, this is currently a one-to-all spawned from a loop. This is good to get things started and correct and we will switch this to a broadcast (one-to-all), and ultimately, a broadcast to a subset of potential workers, e.g., those that had destination mesh vertices found in the source mesh in the previous time step.